### ISSUE TITLE
any way of getting playstore on AOSP13?

### ISSUE THREAD

**Issue by Joduai** [wacko1805/MagiskGapps#15]
Hi
After many hours of trying to get that dang g.play on aosp13, I've found your magisk alpha package for aosp 13.  Unfortunately there's no up to date twrp with sdk >33 for moto G pro,  thus this wouldn't be a thing. 
So I wanted to ask if there's a chance of getting g.play app with your magisk module?
Basic 'strooggle' framework is working nicely,  and since this package contains, and install phonesky in /system/priv-apps I wonder how this could be achieved?

**Comment by wacko1805** [comment]
Sorry i am very confused. The module contains the google play app... Have you installed Magisk, as that is what the module is built for.

**Comment by Joduai** [comment]
Yes, magisk was working as supposed, magiskapps installed sucessfully and I was able to login with my google credentials. But I had no access to play store as there was no shortcut to it nor an app in app drawer.
Now I recall myself that I stepped on somebody's problem with proper rights to 'phonesky' file but i can't find that thread nor remember what it was exactly about.

Nevertheless I spend too much time trying all possible ways, and since I have failed, I suppose I'm at the end of the road with G services on aosp 13. Aurora store seems to work, and the only downside I might find is that banking apps refuse to work.

**Comment by wacko1805** [comment]
Are you on the stock Motorolla firmware or a custom rom?

**Comment by Joduai** [comment]
while using stock I'd never even find this magisk module ;)
I used custom aosp13 for moto g pro/stylus
https://1pwn.ixmoe.com/android/amogus/AOSP-T-Sept/

now after I tried all things I got a small help from another moto user, and moved on to a more stable roms I have no idea that even existed. At least those couldn't be found on sites like xda.
We might close this issue as resolving it is probably impossible.
Thanks for your time, and sorry if I bothered unnecessarily.
